src/utils/apiUtils/timestamps.test.js

```diff
@@ -0,0 +1,277 @@
+import dateFormat from 'dateformat';
+import { expect } from 'chai';
+
+import {
+  formatAndAdjustDateForTimestamp,
+  getAdjustedDateForTimestamp,
+  getEndOfDayTimestampIndex,
+  getStartOfDayTimestampIndex,
+  getTimestampForDay
+} from './timestamps';
+
+describe('getAdjustedDateForTimestamp', function() {
+  it('returns correct date for a timestamp and gmt offset', function() {
+    const gmtoffset = 1000;
+    const timestamp = 2000;
+    const adjustedDateForTimestamp = new Date(3000000);
+
+    expect(getAdjustedDateForTimestamp(gmtoffset, timestamp))
+      .to
+      .deep
+      .equal(adjustedDateForTimestamp);
+  });
+});
+
+describe('formatAndAdjustDateForTimestamp', function() {
+  const gmtoffset = 1000;
+  const timestamp = 2000;
+  const dateAndTimeFormat = 'h:MM TT';
+  const adjustedDateForTimestamp = new Date(3000000);
+
+  it('gets date and time given proper inputs', function() {
+    expect(formatAndAdjustDateForTimestamp(
+      gmtoffset,
+      timestamp,
+      dateAndTimeFormat
+    )).to
+      .equal(dateFormat(adjustedDateForTimestamp, dateAndTimeFormat, true));
```

Comment:
I know all you did was move this test but I'm wondering, is it good to use dateFormat here?  Would it be better to just type out the desired string directly?  I know the latter method would make it much clearer to me what `formatAndAdjustDateForTimestamp` actually does.